### DISCUSSION TITLE
Introduce DepthAI SDK

### DISCUSSION THREAD

**Discussion by VanDavv** [luxonis/depthai#491] [Releases]
DepthAI SDK is a Python package, containing convenience classes and functions that help in most common tasks while using DepthAI API.

Package documentation - https://docs.luxonis.com/projects/sdk/en/latest/
PyPI page - https://pypi.org/project/depthai-sdk/

To install the package, please run the following command

```
pip install depthai-sdk
```

To see this package in action, you can check [the getting started guide](https://docs.luxonis.com/projects/sdk/en/latest/getting_started/) or the following experiments:

- `gen2-people-counter`: https://github.com/luxonis/depthai-experiments/tree/sdk/gen2-people-counter
- `gen2-human-pose`: https://github.com/luxonis/depthai-experiments/tree/sdk/gen2-human-pose
- `gen2-road-segmentation`: https://github.com/luxonis/depthai-experiments/tree/sdk/gen2-road-segmentation

To know more about the DepthAI itself, visit [our documentation](https://docs.luxonis.com/en/latest/) or [our webpage](https://luxonis.com/)

<hr /><em>This discussion was created from the release <a href='https://github.com/luxonis/depthai/releases/tag/v1.0.1-sdk'>Introduce DepthAI SDK</a>.</em>

**Comment by Tria91** [comment]
I just installed depthai-sdk with the command above on my raspberry pi 4, but it show's me the bug "cannot import name 'OakCamera' from 'depthai_sdk'
Please help me. What can I do?
